hummingbot/core/data_type/order_book_tracker_data_source.py

```diff
@@ -50,7 +50,8 @@ async def listen_for_order_book_snapshots(self, ev_loop: asyncio.BaseEventLoop,
         raise NotImplementedError
 
     @abstractmethod
-    async def listen_for_trades(self, ev_loop: asyncio.BaseEventLoop, output: asyncio.Queue):
+    async def \
```

Comment:
Not good formatting here.

Reply:
This is also not intentional. Thanks for pointing out.